CHANGES.rst

```diff
@@ -27,6 +27,7 @@ Fixed
 
 - Fix documentation build error due to deleted link definition to deprecated
   `debops.php5` role repository. [ganto_]
+- Probe if `roundcube__domain` is a string and construct `roundcube__git_checkout` accordingly.  [cultcom]
```

Comment:
You can use

```RST
:envvar:`roundcube__domain`
```

to refer to the variable. It will also hyperlink it :wink: (thanks to @ganto for the hint!)